cpanfile

```diff
@@ -37,6 +65,7 @@ requires 'MooX::Types::MooseLike::Base';  # for Email::Sender::Transport::SMTP w
 requires 'Moose';
 requires 'Moose::Role';
 requires 'Moose::Util::TypeConstraints';
+requires 'MooseX::ClassAttribute';
```

Comment:
Locale::CLDR complains loudly if it not there

Reply:
Ok. Understand it now: It depends on MooX::ClassAttribute, but our classes are Moose classes which means MooX::* gets upgraded to MooseX::* ; we should mention that that's the reason the requirement is upon *us* (instead of a failure from Locale::CLDR).

Reply:
Set in cpanfile